x/incentive/types/params_test.go

```diff
@@ -21,129 +22,130 @@ type ParamTestSuite struct {
 		paramTest{
-			params:     p1,
+			name: "valid",
+			params: types.Params{
+				Active: true,
+				Rewards: types.Rewards{
+					types.Reward{
+						Active:        true,
+						Denom:         "bnb",
+						Reward:        sdk.NewCoin("ukava", sdk.NewInt(10000000000)),
+						Duration:      time.Hour * 24 * 7,
+						TimeLock:      time.Hour * 8766,
+						ClaimDuration: time.Hour * 24 * 14,
+					},
+				},
+			},
 			expectPass: true,
 		},
 		paramTest{
-			params:     p2,
+			name: "duplicate reward",
+			params: types.Params{
+				Active: true,
+				Rewards: types.Rewards{
+					types.Reward{
+						Active:        true,
+						Denom:         "bnb",
+						Reward:        sdk.NewCoin("ukava", sdk.NewInt(10000000000)),
+						Duration:      time.Hour * 24 * 7,
+						TimeLock:      time.Hour * 8766,
+						ClaimDuration: time.Hour * 24 * 14,
+					},
+					types.Reward{
+						Active:        true,
+						Denom:         "bnb",
+						Reward:        sdk.NewCoin("ukava", sdk.NewInt(10000000000)),
+						Duration:      time.Hour * 24 * 7,
+						TimeLock:      time.Hour * 8766,
+						ClaimDuration: time.Hour * 24 * 14,
+					},
+				},
+			},
 			expectPass: false,
 		},
 		paramTest{
-			params:     p3,
+			name: "negative reward duration",
+			params: types.Params{
+				Active: true,
+				Rewards: types.Rewards{
+					types.Reward{
+						Active:        true,
+						Denom:         "bnb",
+						Reward:        sdk.NewCoin("ukava", sdk.NewInt(10000000000)),
+						Duration:      time.Hour * -24 * 7,
+						TimeLock:      time.Hour * 8766,
+						ClaimDuration: time.Hour * 24 * 14,
+					},
+				},
+			},
 			expectPass: false,
 		},
 		paramTest{
-			params:     p4,
+			name: "negative time lock",
+			params: types.Params{
+				Active: true,
+				Rewards: types.Rewards{
+					types.Reward{
+						Active:        true,
+						Denom:         "bnb",
+						Reward:        sdk.NewCoin("ukava", sdk.NewInt(10000000000)),
+						Duration:      time.Hour * 24 * 7,
+						TimeLock:      time.Hour * -8766,
+						ClaimDuration: time.Hour * 24 * 14,
+					},
+				},
+			},
 			expectPass: false,
 		},
 		paramTest{
-			params:     p5,
+			name: "zero claim duration",
+			params: types.Params{
+				Active: true,
+				Rewards: types.Rewards{
+					types.Reward{
+						Active:        true,
+						Denom:         "bnb",
+						Reward:        sdk.NewCoin("ukava", sdk.NewInt(10000000000)),
+						Duration:      time.Hour * 24 * 7,
+						TimeLock:      time.Hour * 8766,
+						ClaimDuration: time.Hour * 0,
+					},
+				},
+			},
 			expectPass: false,
 		},
 		paramTest{
-			params:     p6,
+			name: "zero reward",
+			params: types.Params{
+				Active: true,
+				Rewards: types.Rewards{
+					types.Reward{
+						Active:        true,
+						Denom:         "bnb",
+						Reward:        sdk.NewCoin("ukava", sdk.NewInt(0)),
+						Duration:      time.Hour * 24 * 7,
+						TimeLock:      time.Hour * 8766,
+						ClaimDuration: time.Hour * 0,
```

Comment:
```suggestion
						ClaimDuration: time.Hour * 24 * 14,
```

Test case will fail regardless of Reward's amount due to duplicated invalid ClaimDuration.